graph_creator/views.py

```diff
@@ -42,70 +43,107 @@ def get(self, request, *args, **kwargs):
 class ReceiveData(View):
     """
     Receives and processes data from the remote edx-platform.
-    
+
     If the platform has already registered, a normal data exchange happens.
     Otherwise generates a secret token, (registers)saves it to DB with the edx-platform's incoming URL
-    and sends newly generated token to the edx-platform for further 
+    and sends newly generated token to the edx-platform for further
     data interchange abilities with the server.
     """
 
+    @staticmethod
+    def students_per_country(active_students_amount, students_per_country):
+        """
+        Method calculates amount of students per country.
+
+        Arguments:
+            active_students_amount (int): Count of active students.
+            students_per_country (list): List of dictionaries, where one of them is country-count accordance.
+                                         Amount of students without country is empty.
+
+        Returns:
+            students_per_country (list): List of dictionaries, where one of them is country-count accordance.
+                                         Amount of students without country has calculated.
+        """
+
+        students_per_country[0]['count'] = active_students_amount - sum(
```

Comment:
Very hard to understand logic, can you please rewrite in more simple way? 
OR add good example to docstring